templates/test.yml

```diff
@@ -0,0 +1,89 @@
+# Copyright 2024 Defense Unicorns
+# SPDX-License-Identifier: AGPL-3.0-or-later OR LicenseRef-Defense-Unicorns-Commercial
+
+spec:
+  inputs:
+    runsOn:
+    flavor:
+    type:
+    artifact_dir:
+      default: .ci_artifacts
+    reports-path:
```

Comment:
same comment on input casing

Reply:
Fixed.

Reply:
Ended up refactoring inputs in general to be dashes instead of camelCase throughout the GitLab CI components so it's all consistent.